pepper-apis/src/main/java/org/broadinstitute/ddp/util/DataExporterCli.java

```diff
@@ -149,13 +155,56 @@ private void initExporter() {
         exporter = new DataExporter(cfg);
     }
 
-    private void runEsExport(String studyGuid, Set<String> participantGuids, boolean exportStructuredDocument) {
+    private void runAllElasticExports(String studyGuid) {
+        long begin = System.currentTimeMillis();
         TransactionWrapper.useTxn(handle -> {
             System.out.println("[export] warming up caches...");
+
             StudyDto studyDto = handle.attach(JdbiUmbrellaStudy.class).findByStudyGuid(studyGuid);
+            DataExporter.fetchAndCacheAuth0Emails(handle, studyGuid, handle
+                    .select("select auth0_user_id from user")
+                    .mapTo(String.class)
+                    .stream()
+                    .collect(Collectors.toSet()));
+
+            System.out.println("[export] extracting participants...");
+            long start = System.currentTimeMillis();
+            List<Participant> participants = exporter.extractParticipantDataSet(handle, studyDto);
+            long elapsed = System.currentTimeMillis() - start;
+            System.out.println(String.format("[export] took %d ms (%.2f s)", elapsed, elapsed / 1000.0));
 
-            // Warm up the activity cache
-            exporter.extractActivities(handle, studyDto);
+            System.out.println("[export] starting activity definitions export...");
+            start = System.currentTimeMillis();
+            List<ActivityExtract> activities = exporter.exportActivityDefinitionsToElasticsearch(handle, studyDto, cfg);
+            elapsed = System.currentTimeMillis() - start;
+            System.out.println(String.format("[export] took %d ms (%.2f s)", elapsed, elapsed / 1000.0));
+
+            System.out.println("[export] starting structured json export...");
+            start = System.currentTimeMillis();
```

Comment:
Looks like we can extract a private method  and avoid some code duplication. Same statements are invoked for 3 indices and probably same from other method too. 
Not lot of statements though.

Reply:
I think it's fine -- it's just timing statements. I'll like to keep this the same as what's merge into `develop`. We can refactor later if we'll like.

Reply:
ok. If you want to revisit it later, thats fine.